addons/l10n_tw_edi_ecpay_website_sale/static/src/js/website_sale_address.js

```diff
@@ -0,0 +1,18 @@
+/** @odoo-module **/
+import websiteSaleAddress from "@website_sale/js/address";
+
+websiteSaleAddress.include({
+    events: Object.assign({}, websiteSaleAddress.prototype.events, {
+        "input input[name='company_name']": "_onChangeCompanyName",
+    }),
+
+    _onChangeCompanyName(ev) {
+        const companyName = this.addressForm.o_company_name.value;
+
+        if (companyName) {
+            this._hideInput("l10n_tw_edi_require_paper_format");
+        } else {
+            this._showInput("l10n_tw_edi_require_paper_format");
+        }
+    },
```

Comment:
When editing an address where the company name is set, the 'Request a paper copy' will be visible (until I edit the company name again), you probably should find some way to hide it in this case.